index.js

```diff
@@ -1,20 +1,21 @@
 /* global Blob */
 
-var once = require('once')
-
-module.exports = function getBlob (stream, mimeType, cb) {
```

Comment:
Don't remove the name of the function. This is useful in stack traces when debugging.